client/wfm/src/workorder/WorkOrder.ts

```diff
@@ -28,17 +33,16 @@ export interface ProcessInstance {
   /**
    * Longer description for this {@link ProcessInstance}
    */
-  comment: string;
+  summary: string;
```

Comment:
Actually that's the data we do not want here as having this will imply that user workorders will have location etc. Can we use some generic data field like `attributes` that users can use to post any content. We can adjust UI to reflect to this and this data will be actually demo specific. 

💯  I'm really against having this in framework.